src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Rewriter/MarkdownDocumentVisitor.cs

```diff
@@ -26,6 +32,11 @@ public void Visit(MarkdownDocument document)
 
             // rewrite root node of AST
             document = _rewriter.Rewrite(document) as MarkdownDocument;
+            if (document == null)
+            {
+                throw new Exception($"The rewrite result can't be null.");
```

Comment:
A more meaningful exception such as `InvalidOperationException`? And also provide more information in error message